courses/management/commands/manage_certificates.py

```diff
@@ -143,6 +150,11 @@ def handle(self, *args, **options):  # pylint: disable=too-many-locals
             if override_grade and not is_grade_valid(override_grade):
                 raise CommandError("Invalid value for grade. Allowed range: 0.0 - 1.0.")
 
+            if override_grade and not letter_grade:
```

Comment:
We needed to pass the letter grade to `override_user_grade` after line 186.
<img width="1772" alt="Screen Shot 2023-07-24 at 5 38 06 PM" src="https://github.com/mitodl/mitxonline/assets/52656433/5ed6070e-1bf1-47b7-b32a-81ddeff756dc">